contacts/templates/contacts/contact.html

```diff
@@ -0,0 +1,18 @@
+{% load staticfiles %}
+
+<link rel="stylesheet" type="text/css" href="{% static 'contacts/style.css' %}" />
```

Comment:
css/js imports should be in header of the html page.

Reply:
done